tasks/release.py

```diff
@@ -117,3 +121,139 @@ def _find_v6_tag(ctx, v7_tag):
             print("Found v6 tag '{}'".format(v6_tag))
 
     return v6_tag
+
+
+def _is_version_higher(version_1, version_2):
+    if not version_2:
+        return True
+    if version_1["major"] < version_2["major"]:
+        return False
+    if version_1["minor"] < version_2["minor"]:
+        return False
+    if version_1["patch"] < version_2["patch"]:
+        return False
+    if version_1["rc"] < version_2["rc"]:
+        return False
+    return True
+
+
+def _create_version_dict_from_match(match):
+    groups = match.groups()
+    version = {
+        "major": int(groups[0]),
+        "minor": int(groups[1]),
+        "patch": int(groups[2]),
+        "rc": 0
```

Comment:
```suggestion
        "rc": int(groups[4]) if groups[4] else 0
```
and you can then remove the conditional below.